README.md

```diff
@@ -138,3 +138,90 @@ c, err := expect.NewConsole(expect.WithStdout(out))
 assert.Nil(t, err)
 defer c.Close()
 ```
+
+## Undocumented Features and Flags
+
+### Global Flags
+
+#### Using `--realm-url` and `--atlas-url`
+
+At some point, you may wish to configure which Realm and Atlas servers the CLI is connecting to.  By default, these URLs point to the Production instances of Realm and Atlas, which should be sufficient for most use cases.
+
+However, you can set `--realm-url` and `--atlas-url` to any URLs you know a respective server instance is running at.  If you are configuring one, chances are you'll want to configure both of these.
```

Comment:
wouldn't you _always_ need to configure both? not sure how realm dev would work with cloud qa for example

Reply:
the one a-typical scenario thats come up in practice is local+dev, but the technicality still remains that you'd be setting both of those at once.

also, if you didn't heed this advice, then you'd read it and say "oh yeah, those chances were right!  but now I only need to set one" :)

Reply:
I wonder if you could ever rig up a local Realm + prod Atlas env...